### PR TITLE
media-libs/zita-convolver: version bump to 4.0.3

### PR DESCRIPTION
**Pull request by puleglot**
* EAPI=7
* add subslot

Closes: https://bugs.gentoo.org/684534
Package-Manager: Portage-2.3.66, Repoman-2.3.11
Signed-off-by: Alexander Tsoy <alexander@tsoy.me>

### CODE REVIEW

File: media-libs/zita-convolver/zita-convolver-4.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="C++ library implementing a real-time convolution matrix"
+HOMEPAGE="http://kokkinizita.linuxaudio.org/linuxaudio/"
```
Comment (a17r) [inline media-libs/zita-convolver/zita-convolver-4.0.3.ebuild]: seems you can go `https` there

File: media-libs/zita-convolver/zita-convolver-4.0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+SRC_URI="http://kokkinizita.linuxaudio.org/linuxaudio/downloads/${P}.tar.bz2"
```
Comment (a17r) [inline media-libs/zita-convolver/zita-convolver-4.0.3.ebuild]: check here as well

File: media-libs/zita-convolver/zita-convolver-4.0.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+LICENSE="GPL-3+"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="sci-libs/fftw:3.0="
+DEPEND="${RDEPEND}"
+BDEPEND=""
```
Comment (a17r) [inline media-libs/zita-convolver/zita-convolver-4.0.3.ebuild]: just drop it if it's empty

File: media-libs/zita-convolver/zita-convolver-4.0.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+PATCHES=( "${FILESDIR}"/${PN}-4.0.3-makefile.patch )
+
+src_compile() {
+	emake -C source CXX="$(tc-getCXX)"
+}
+
+src_install() {
+	local myemakeargs=(
```
Comment (a17r) [inline media-libs/zita-convolver/zita-convolver-4.0.3.ebuild]: configure in `src_configure`, install in `src_install`

File: media-libs/zita-convolver/zita-convolver-4.0.3.ebuild [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
```
Comment (a17r) [inline media-libs/zita-convolver/zita-convolver-4.0.3.ebuild]: I don't see it used?

Comment (puleglot) [reply media-libs/zita-convolver/zita-convolver-4.0.3.ebuild]: tc-getCXX?

File: media-libs/zita-convolver/zita-convolver-4.0.3.ebuild (hunk 4 above)
Comment (puleglot) [reply media-libs/zita-convolver/zita-convolver-4.0.3.ebuild]: This is a Makefile-based build system. And all these argument only meaningful in `src_install` phase. So I don't see what's wrong here.

**Comment by mgorny** [comment]
LGTM. I think we can assume maintainer timeout by now, so I'm going to merge it. If you're interested in maintaining it, please submit PR adding yourself as the maintainer.

**Pull request closed without merging**